--- origsrc/gtk+-3.0.8/docs/reference/gdk/Makefile.am	2011-02-11 14:24:40.000000000 -0600
+++ src/gtk+-3.0.8/docs/reference/gdk/Makefile.am	2011-04-06 05:26:49.922106200 -0500
@@ -33,7 +33,7 @@ IGNORE_HFILES= 			\
 
 # Extra files to add when scanning (relative to $srcdir)
 EXTRA_HFILES=			\
-	../../../gdk/x11/gdkx.h
+	$(top_srcdir)/gdk/x11/gdkx.h
 
 # CFLAGS and LDFLAGS for compiling scan program. Only needed
 # if $(DOC_MODULE).types is non-empty.
